clients/gen/python.sh

```diff
@@ -25,7 +25,7 @@ readonly CLEANUP_DIRS
 # shellcheck source=./clients/gen/common.sh
 source "${CLIENTS_GEN_DIR}/common.sh"
 
-VERSION=1.0.0
+VERSION=2.0.1
```

Comment:
Hi @msumit , I may (or for sure) missed something ;-)

May you help me understand why it's 1.0.0 -> 2.0.1 here? It should be 2.0.0 -> 2.0.1 here to my understanding. Was the `VERSION` missed to be updated when we released apache-airflow-client 2.0.0?

But overall LGTM

Reply:
Hi @XD-DENG, yup seems like we had missed upgrading the version here.